ts/types.ts

```diff
@@ -169,18 +174,18 @@ const DefaultIntrinsicsList = [
   "Promise",
 ] as const
 
-/**
- * Work in progress.
- */
-export const DefaultIntrinsics = Symbol("DefaultIntrinsics")
-
 export interface ContextOptions {
   /**
-   * What built-in objects and language features to enable?
+   * What built-in objects and language features to enable.
+   *
    * If unset, the default intrinsics will be used.
-   * To omit all intrinsics, pass an empty array.
+   *
+   * The base objects intrinsics will always be enabled,
+   *  but to omit all additional intrinsics, pass an empty array.
+   * 
+   * To be able to use `evalCode` you must enable the "Eval" intrinsic.
    */
-  intrinsics?: PartiallyImplemented<Intrinsic[]> | typeof DefaultIntrinsics
+  intrinsics?: Intrinsic[]
```

Comment:
Ideally we can accept DefaultIntrinsics or similar if a user wants to add stuff on top of the defauls

Reply:
Hmm, I'm not sure the best way to support that. 
What if we export the default intrinsic list and the user can extend it on their side if desired?

Reply:
The initial implementation had a stand-in symbol DefaultIntrinsics which if present should add all the default intrinsics. Giving the array works too and is certainly easier to understand.